### PR TITLE
[R2] Header

### PR DESCRIPTION
**Pull request by g-stamatis**
https://taskman.eionet.europa.eu/issues/147798

https://62713c57bcbdf4004a3d99b8-qrpcgnijpk.chromatic.com/?path=/story/layout-header--default

### CODE REVIEW

**Comment by g-stamatis** [comment]
@eea-jenkins build storybook

**Comment by tarantilis** [comment]
Didn't manage to use a "helper type solution" with callback as we had for other components, had hard time with the refs and triggers.
Since we had one occurrence with this bug, I used onClick function to stop propagation.
Works well, if there is a need to find a more global way to prevent multiple dropdowns we look at it again

File: theme/themes/eea/extras/header.less [hunk 1]
```diff
@@ -415,14 +433,22 @@
...
-      width: 19px;
+    .divider.text {
+      font-size: @font-size-1;
+      line-height: @font-lineheight-00;
     }
 
-    .menu {
-      top: @tabletLanguageMenuTop;
+    img {
+      width: 18px;
```
Comment (ichim-david) [inline theme/themes/eea/extras/header.less:451]: @tarantilis add the values in variables

**Review by ichim-david — CHANGES_REQUESTED** [review]

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

**Review by ichim-david — APPROVED** [review]
@tarantilis  I will merge this pull request however I think another pull request will be needed after we clarify the font size of the language dropdown

**Pull request merged**